_tuning-your-cluster/availability-and-recovery/remote-store/index.md

```diff
@@ -104,9 +104,65 @@ If the Security plugin is enabled, a user must have the `cluster:admin/remotesto
 
 You can use remote-backed storage for the following purposes:
 
-- To restore red clusters or indexes
-- To recover all data up to the last acknowledged write, regardless of replica count, if `index.translog.durability` is set to `request`
+- To restore red clusters or indexes.
+- To recover all data up to the last acknowledged write, regardless of replica count, if `index.translog.durability` is set to `request`.
 
+## Benchmarks
+
+The OpenSearch team has run remote store using multiple workload options available within the [OpenSearch Benchmark (OSB)](https://opensearch.org/docs/latest/benchmark/index/) tool. This section summarizes the benchmark results for the following OSB workloads: 
+
+- [StackOverflow](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/so)
+- [HTTP logs](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/http_logs)
+- [NYC taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis),
+
+Each workload was tested using multiple cluster configurations. Indexed data was added to each cluster using varying numbers of bulk indexing clients. This simulates a high level of requests made to the cluster concurrently. 
```

Comment:
This line conveys a different meaning. Can we reword this as - `Each workload was tested against multiple configurations of bulk indexing clients which simulates varying degree of concurrency of requests.`